playbooks/openstack/configuration.md

```diff
@@ -856,15 +859,15 @@ openshift_openstack_num_nodes: 8  # 5 existing and 3 new
 
 ### 2. Scale the Cluster
 
-Next, run the appropriate playbook - either 
+Next, run the appropriate playbook - either
 `openshift-ansible/playbooks/openstack/openshift-cluster/master-scaleup.yml`
 for master nodes or
 `openshift-ansible/playbooks/openstack/openshift-cluster/node-scaleup.yml`
 for other nodes. For example:
 
 ```
 $ ansible-playbook --user openshift \
-  -i openshift-ansible/playbooks/openstack/inventory.py \
+  -i openshift-ansible/playbooks/openstack/scaleup_inventory.py \
```

Comment:
Is there any other case where we should use the `scaleup_inventory.py` file? Such as `configure-new-nodes.yml` or `provision_resources.yml`?

Reply:
If you don't rely on the `masters`, `etcd` etc. groups explicitly yourself, you can use the `scaleup_inventory` pretty much everywhere. All the playbooks under `openstack` should work with it. I mean, that's how it is now without this PR.

I've renamed it to let people use inventory.py as before (so they don't have to update their groups like you had to -- again sorry for not doing that in the first place).

And also for running the standalone non-openstack playbooks in this repo e.g. `playbooks/openshift-logging/config.yml` and similar.